.github/workflows/build-pyodide-package.yml

```diff
@@ -0,0 +1,108 @@
+name: Build pyodide arbor
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      PYODIDE_VERSION: 0.22.0a3
+      PYODIDE_RELEASE_URL: "https://github.com/pyodide/pyodide/releases/download"
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+        path: arbor_src
+    - name: get-xbuildenv
+      run: |
+        XBUILDENV_BASENAME=xbuildenv-${PYODIDE_VERSION}.tar.bz2
+        wget -nc "${PYODIDE_RELEASE_URL}/${PYODIDE_VERSION}/${XBUILDENV_BASENAME}"
+        tar -xvf ${XBUILDENV_BASENAME}
+        rm ${XBUILDENV_BASENAME}
+    - name: read-makefile
+      run: |
+        make -f xbuildenv/pyodide-root/Makefile.envs .output_vars | grep 'PY' | tee -a $GITHUB_ENV
+    - name: get-emsdk
+      run: |
+        wget https://github.com/emscripten-core/emsdk/archive/refs/heads/main.zip
+        unzip main.zip
+        rm main.zip
+    - name: setup-emsdk
+      run: |
+        ./emsdk-main/emsdk install ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+        ./emsdk-main/emsdk activate ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+    - name: build-modcc-host
+      run: |
+        mkdir -p arbor_src/build_modcc_host
+        cd arbor_src/build_modcc_host
+        cmake .. -DARB_USE_BUNDLED_LIBS=ON
+        make modcc -j 8
+    - name: build-libarbor-emscripten
+      run: |
+        ABS_TOOLCHAIN=$(realpath emsdk-main/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake)
+        ABS_MODCC=$(realpath arbor_src/build_modcc_host/bin/modcc)
+        sed -i.bak -E \
```

Comment:
Why is this needed?

Reply:
Because Random123 refuses to build on any non-standard platform sadly

This is the code that we delete:

```c++
#if !defined(__x86_64__) && !defined(__i386__) && !defined(__powerpc__) && !defined(__arm__) && !defined(__aarch64__) && !defined(__s390x__)
#  error "This code has only been tested on x86, powerpc and a few arm platforms."
#include <including_a_nonexistent_file_will_stop_some_compilers_from_continuing_with_a_hopeless_task>
{ /* maybe an unbalanced brace will terminate the compilation */
 /* Feel free to try the Random123 library on other architectures by changing
 the conditions that reach this error, but you should consider it a
 porting exercise and expect to encounter bugs and deficiencies.
 Please let the authors know of any successes (or failures). */
#endif
```

So technically we're doing something very wrong, but the code seems to work fine for now.. Maybe we should test the random numerics on wasm :)